bundle/manifests/rhdh.redhat.com_orchestrators.yaml

```diff
@@ -328,6 +347,10 @@ spec:
                         description: Namespace determines the namespace where the
                           operator should be deployed. Defaults to rhdh-operator
                         type: string
+                      source:
+                        default: redhat-operators
+                        description: Source captures the name of the catalog source
+                        type: string
```

Comment:
Shouldn't the sourceName below be removed?

Reply:
oh, no changes to the schema file in this PR.

Reply:
It should be `sourceName`. Let me find out why it hasn't been replace.